README.md

```diff
@@ -52,7 +52,7 @@ from one program to another                # cmd + tab
 from one window to another (same program)  # cmd + `
 ```
 
-### Sublime Text
+### Text Editor (Sublime Text or VS Code with Sublime Text Keymap)
```

Comment:
Should we leave only "Text Editor" as the title and add a `<p>` to specify both environments ❓